examples/opf/experiments/multistep/base/permutations_simple_3.py

```diff
@@ -58,7 +59,7 @@
   'predictedField': 'field2',
   'predictionSteps': [1,3],
   
-  'dataSource': 'file://%s' % (os.path.join(os.environ['NUPIC'], 'examples', 'opf', 
+  'dataSource': 'file://%s' % (os.path.join(NUPIC_ROOT, 'examples', 'opf', 
```

Comment:
`examples` is in the root of the repo which is a level above `NUPIC_ROOT`. And pkg_resources would probably be better here but up to you